docs/data-sources/placement_group.md

```diff
@@ -0,0 +1,47 @@
+---
+page_title: "Linode: linode_placement_group"
+description: |-
+  Provides details about a Linode placement group.
+---
+
+# Data Source: linode\_placement\_group
+
+`linode_placement_group` provides details about a Linode placement group.
+
+## Example Usage
+
+The following example shows how the resource might be used to obtain additional information about a Linode placement group.
+
+```hcl
+data "linode_placement_group" "pg" {
+  id = 12345
+}
+```
+
+## Argument Reference
+
+* `id` - (Required) The ID of the Placement Group.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `label` - (Required) The label of the Placement Group. This field can only contain ASCII letters, digits and dashes.
+
+* `region` - (Required) The region of the Placement Group.
+
+* `affinity_type` - (Required) The affinity policy to use when placing Linodes in this group.
+
+* `is_strict` - (Optional) Whether Linodes must be able to become compliant during assignment. (Default `true`)
+
+* `is_compliant` - (Optional) Whether all Linodes in this group are currently compliant with the group's affinity policy.
```

Comment:
Oops, I missed this during my original review but I think the `(Required)` and `(Optional)` prefixes can be dropped from these docs 🙂